apps/generator/test/test-project/package.json

```
@@ -1,6 +1,7 @@
 {
     "description": "Test project. More info in the readme.",
     "scripts": {
+        "test": "jest",
```

Biome (1.9.4)

[error] The key test was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)